providers/frame/zombienet.ts

```diff
@@ -58,6 +63,23 @@ export class ZombienetProvider extends FrameProxyProvider {
   zombienet(configPath: string): Promise<Network> {
     return this.zombienetMemo.run(configPath, async () => {
       const tmpDir = Deno.makeTempDirSync({ prefix: `capi_zombienet_` })
+      const watcher = Deno.watchFs(tmpDir)
+      const closeWatcher = () => {
+        if (typeof Deno.resources()[watcher.rid] === "number") watcher.close()
```

Comment:
Why do we need to do this?

Reply:
If we don't, then we're at risk of getting a bad resource ID error (in cases where we attempt to close the watcher twice, such as successful resolution of the the config, and then aborting of the server)

Reply:
I think it would be cleaner to have something like
```ts
let closed = false
const closeWatcher = () => {
  if(closed) return
  closed = true
  watcher.close()
}
```
instead of relying on the resources

Reply:
You need to remove the explicit `closeWatcher()` before the `return JSON.parse`